ios-trivia-game/API/FirebaseClient.swift

```diff
@@ -452,7 +460,7 @@ class FirebaseClient {
     // gets invites for a user
     func getInvitesFor(userId: String, complete: @escaping (FIRDataSnapshot) -> (), onError: ((Error?) -> ())?) {
         let path = "\(Constants.INVITE_TABLE_NAME)"
-        ref.child(path).queryOrdered(byChild: "guest_id").queryEqual(toValue: userId).observe(.value, with: { (snapshot) in
+        ref.child(path).queryOrdered(byChild: "guest_id").queryEqual(toValue: userId).observeSingleEvent(of: .value, with: { (snapshot) in
```

Comment:
Can we keep this as a listener? The notification view relies on this to show the pop-up bar.